host/windows/syscall.c

```diff
@@ -3124,3 +3125,34 @@ int oe_syscall_nanosleep_ocall(struct oe_timespec* req, struct oe_timespec* rem)
     _set_errno(0);
     return 0;
 }
+
+/*
+**==============================================================================
+**
+** clock_nanosleep():
+**
+**==============================================================================
+*/
+
+int oe_syscall_clock_nanosleep_ocall(
+    oe_clockid_t clockid,
+    int flag,
+    struct oe_timespec* req,
+    struct oe_timespec* rem)
+{
+    /* Validate input */
+    if (flag != 0 || clockid == CLOCK_THREAD_CPUTIME_ID)
```

Comment:
Your response to my incorrect changelog suggestion made me realize that the test here is incorrect.
include/openenclave/internal/time.h only defines values 0-9.
If the caller passes clockid = 10 (or 42 or MAX_INT), this will accept it, but Linux will fail it.
Hence this should fail for invalid clockid values.

Reply:
I should have thought about that. Thank you for pointing it out.